conf/airframes/examples/ardrone2_FPguided_gazebo.xml

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="ardrone2">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="ardrone2"/>
+
+    <target name="nps" board="pc">
+      <module name="fdm" type="gazebo"/>
+    </target>
+
+    <define name="USE_SONAR" value="TRUE"/>
+
+	<!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="ardrone2"/>
+    <module name="imu" type="ardrone2"/>
+    <!-- gps: "ublox" or change to "sirf" for usage with parrot flight recorder -->
+    <module name="gps" type="ublox"/>
+    <module name="stabilization" type="int_quat"/>
+    <module name="ahrs" type="int_cmpl_quat"/>
+    <module name="ins" type="extended"/>
+  </firmware>
+
+  <modules main_freq="512">
```

Comment:
Can you the modules from this `modules` node inside the `firmware` node ? I recommend to stop using it as it is planed to be removed some day.